File: src/main/java/meetme/conversations/ConversationsRepository.java
package meetme.conversations;

import org.springframework.data.repository.CrudRepository;

import java.util.List;

public interface ConversationsRepository extends CrudRepository<Conversations, String> {

    List<Conversations> findByOtherUserIdOrLoggedInUserId(String loggedInUserId, String inUserId);
    List<Conversations> findByThreadId(String loggedInUserId);


}

File: src/main/java/meetme/conversations/ConversationsRepository.kt
package meetme.conversations

import org.springframework.data.repository.CrudRepository

interface ConversationsRepository : CrudRepository<Conversations, String> {

    fun findByFirstUserIdOrSecondUserId(loggedInUserId: String, inUserId: String): List<Conversations>
    fun findByThreadId(loggedInUserId: String): List<Conversations>

}
